js/charts/body-mass-index-chart.js

```diff
@@ -35,6 +35,9 @@
 
         _get_dataPoints : function()
         {
+            if (GC.App.getPrimaryChartType() === "FENTON") {
```

Comment:
Can you use `noDataPoints` and set it to `true`? and pass `noDataPoints` to the `.call()`.